wordcloud/views.py

```diff
@@ -0,0 +1,26 @@
+import os
+
+from django.conf import settings
+from django.http import HttpResponse
+from django.utils import simplejson
+from django.views.decorators.cache import cache_page
+
+from .wordcloud import popular_words
+
+
+@cache_page(60*60*4)
+def wordcloud(request, max_entries=30):
+    """ Return tag cloud JSON results"""
+    cache_path = settings.WORDCLOUD_CACHE_PATH
+    if os.path.exists(cache_path):
+        response = HttpResponse()
+        response['Content-Type'] = 'application/json'
+        response['X-Sendfile'] = cache_path.encode('utf-8')
```

Comment:
I think this will also need `libapache2-mod-xsendfile` added to `conf/packages` and you'll need to add `XSendFile        On` and set `XSendFilePath` in the Apache configuration. For trying it out, if you don't have a local Apache, you could create a dev vhost on firefly or do it in a Vagrant box.

I think I mentioned this before, but I think I'd put this generated file in a subdirectory under `MEDIA_ROOT`.